rxbinding-design/build.gradle

```diff
@@ -27,7 +27,7 @@ android {
   defaultConfig {
     minSdkVersion rootProject.ext.minSdkVersion
 
-    testInstrumentationRunner 'com.jakewharton.rxbinding.RxBindingTestRunner'
+    testInstrumentationRunner 'com.jakewharton.rxbinding2.RxBindingTestRunner'
```

Comment:
RxBindingTestRunner hasn't been moved yet.
It is just the next thing that we have to change to actually run tests.